improver_tests/acceptance/test_threshold.py

```diff
@@ -216,19 +210,3 @@ def test_vicinity_masked(tmp_path):
     ]
     run_cli(args)
     acc.compare(output_path, kgo_path)
-
-
-def test_landmask_without_vicinity():
```

Comment:
I think we should still raise this error, but from `BasicThreshold.process` where we already check for these items separately. It is reasonable to move the test from acceptance to unit though.

Reply:
Added into the plugin.